ctrlf.el

```diff
@@ -699,7 +704,10 @@ later (this should be used at the end of the search)."
             (if (and (not skip-search)
                      (ctrlf--search input :bound 'wraparound))
                 (progn
-                  (goto-char (match-beginning 0))
+                  (goto-char (or
+                              (and ctrlf-go-to-end-of-match
+                                   (match-end 0))
+                              (match-beginning 0)))
```

Comment:
What I'm suggesting is the following change, which I think is easier to understand than the combination of `and` and `or` (which is, admittedly, equivalent):

```suggestion
                  (goto-char (if ctrlf-go-to-end-of-match
                                 (match-end 0)
                              (match-beginning 0)))
```

Reply:
Yeah, pretty much